### PR TITLE
Splits Project into Core and ApplicationUI

### PR DESCRIPTION
**Pull request by JLLeitschuh**
This allows us to build the project in headless mode

### CODE REVIEW

**Comment by ThomasJClark** [comment]
I don't know if this is a good approach or not, but let's put headless mode off until serialization is working so we can work on the tasks we have due this week. We have a schedule to stick to!

**Comment by JLLeitschuh** [comment]
This is how gradle is designed. This is how it is supposed to be implemented.

**Comment by ThomasJClark** [comment]
I didn't say there's anything wrong with it. Just that if it's gonna block all the stuff we want to get done now, and its not gonna work until we have serialization, we should wait. The way headless mode works is pretty important, and we should meet to come up with something.

File: core/src/main/java/edu/wpi/grip/core/operations/PythonScriptOperation.java [hunk 1]
```diff
@@ -30,16 +33,16 @@
...
  * and returns the values for the outputs.  It can return a single value if there's one output, or a sequence type for
  * any number of values.
- *
+ * <p>
  * <pre>{@code
  * def perform(a, b):
  * return a + b
  * }</pre>
- *
+ * <p>
```
Comment (JLLeitschuh) [inline core/src/main/java/edu/wpi/grip/core/operations/PythonScriptOperation.java]: Inserted by autoformat. This was not me

**Pull request closed without merging**